go-controller/pkg/ovn/master.go

```diff
@@ -1378,34 +1326,41 @@ func (oc *Controller) syncNodesPeriodic() {
 		nodeNames = append(nodeNames, node.Name)
 	}
 
-	chassisList, err := oc.ovnSBClient.ChassisList()
+	chassisList, err := libovsdbops.FindChassis(oc.sbClient)
 	if err != nil {
 		klog.Errorf("Failed to get chassis list: error: %v", err)
```

Comment:
how about err == libovsdbclient.ErrNotFound  ? maybe we do not need to log an error in that case, still.

Reply:
Agreed will update